packages/gestalt/src/Toast/PrimaryAction.tsx

```diff
@@ -39,6 +39,7 @@ export default function PrimaryAction({ accessibilityLabel, label, size = 'lg',
   return (
     <Button
       accessibilityLabel={accessibilityLabel}
+      color="white"
```

Comment:
@AlbertCarreras This might be wrong -- The default color for Button is changing to white/dark on VR right? Because this 'white' prop is meant to be used in front of dark images, and will not work with dark mode.
I think the right approach is to leave as it was before and make the default white/black when we get to implementing Button VR.

Reply:
can we add white "conditionally to  VR"

Reply:
color={ experiment ?  'white' : undefined}

Reply:
DONE